discovery-provider/src/queries/get_trending_tracks.py

```diff
@@ -29,18 +32,40 @@ def make_trending_cache_key(
 def generate_unpopulated_trending(
     session, genre, time_range, strategy, limit=TRENDING_LIMIT
 ):
-    trending_tracks = generate_trending(session, time_range, genre, limit, 0, strategy)
-
-    track_scores = [
-        strategy.get_track_score(time_range, track)
-        for track in trending_tracks["listen_counts"]
-    ]
-    # Re apply the limit just in case we did decide to include more tracks in the scoring than the limit
-    sorted_track_scores = sorted(track_scores, key=lambda k: k["score"], reverse=True)[
-        :limit
-    ]
-    track_ids = [track["track_id"] for track in sorted_track_scores]
+    top_listen_tracks_subquery = session.query(AggregateIntervalPlay.track_id)
```

Comment:
Why are we getting tracks by listens before their trending scores?

Reply:
This is just a subquery passed into the `TrackTrendingScore` query below to keep parity with the existing trending strategy